### DISCUSSION TITLE
Using a plugin to extra searchable parameters to Purchase Orders

### DISCUSSION THREAD

**Discussion by steves2j** [inventree/InvenTree#3660] [General]
Hi,

Can a plugin add extra parameters to various parts of the Inventree? Specifically I would like to add a Shipment Tracking number to the purchase order so that I can track incoming packages. And be able to search for it.

Steve

**Comment by matmair** [comment]
@steves2j a plugin can add data to a few models via the metadata field. You will have to build the search endpoint yourself though.

**Comment by steves2j** [comment]
So no way to hook into the System search tool via a plugin?

Stephen Edwards
+6421913662
________________________________

**Reply by matmair** [reply]
@steves2j not currently. We are open for PRs.
